package.json

```diff
@@ -46,6 +46,7 @@
   "devDependencies": {
     "@open-draft/test-server": "^0.2.3",
     "@ossjs/release": "^0.3.0",
+    "@types/debug": "^4.1.7",
```

Comment:
Are you getting some typescript errors due to this package missing? Could you please share those?

Reply:
I got this error in a CI build.
https://github.com/mswjs/http-middleware/runs/7314219196?check_suite_focus=true

see https://github.com/mswjs/interceptors/issues/257 for the detail.

Reply:
sure, will update the PR when new `@mswjs/interceptors` is published.

Reply:
I'm not sure if it is added as peerDependencies or not.